security/cert_manager_operator/index.adoc

```diff
@@ -20,9 +20,15 @@ include::modules/cert-manager-request-methods.adoc[leveloffset=+1]
 //Supported versions
 include::modules/cert-manager-supported-versions.adoc[leveloffset=+1]
 
+//FIPS compliant support
+include::modules/cert-manager-fips-support.adoc[leveloffset=+1]
+
+include::modules/cert-manager-fips-procedure.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 [id="cert-manager-operator-about_additional-resources"]
 == Additional resources
 
 * link:https://cert-manager.io/docs/[cert-manager project documentation]
```

Comment:
Also, you can consider including the following FIPS related pages:
- https://docs.openshift.com/container-platform/4.14/security/container_security/security-compliance.html
- https://docs.openshift.com/container-platform/4.14/installing/installing-fips.html#installing-fips-mode_installing-fips